src/test/scala/cromwell/binding/WdlExpressionSpec.scala

```diff
@@ -62,7 +61,7 @@ class WdlExpressionSpec extends FlatSpec with Matchers {
   it should "Int + String = String" in {
     constEval(""" 1 + "String" """) shouldEqual WdlString("1String")
   }
-  it should "Int + float = float" in {
+  it should "Int + Float = Float" in {
```

Comment:
Thanks for fixing these, I saw them when I was in here and meant to clean them up but forgot.  :smile: